packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Helper method to determine which comparison results are relevant if the screenshot assertion
+ * fails.
+ *
+ * @param {Object} screenshot - The comparison result
+ * @param {boolean} screenshot.isSameDimensions - If the latest screenshot was the same size as the reference screenshot.
+ * @param {boolean} screenshot.isWithinMismatchTolerance - If the latest screenshot was within the mismatch tolerance.
+ * @param {boolean} screenshot.isExactSameImage - If the latest screenshot matched the reference screenshot exactly, i.e. 0% mismatch.
+ * @param {Number} screenshot.misMatchPercentage - The mismatch percentage when comparing the latest screenshot to the reference screenshot.
+ * @returns {String} - A message describing the status of the screenshot.
+ */
+function getComparisonResult(screenshot) {
+  const { misMatchPercentage, isSameDimensions } = screenshot;
+  const relevantInformation = {};
+
+  if (!isSameDimensions) {
+    relevantInformation.isSameDimensions = isSameDimensions;
```

Comment:
Curious about this logic assignment. Assign isSameDimensions only if `false`? True doesn't getting added into the property list?

Reply:
Restructured the error message to make it fore human friendly to read based on the suggestion below. https://github.com/cerner/terra-toolkit/pull/545/commits/9933873194e2d7e6d395853f313cdd4cf89d1212